### PR TITLE
Added bug alias as feature

### LINKED ISSUE #127: Add bug alias as a feature
Not the alias by itself, but something like True if `CVE` is in bug['alias'], False otherwise.

### PR DESCRIPTION
**Pull request by Aftaab99**
Fix #127

### CODE REVIEW

File: bugbug/bug_features.py [hunk 1]
```diff
@@ -183,6 +183,14 @@ def __call__(self, bug):
         return bug['priority']
 
 
+class bug_has_alias(object):
+    def __call__(self, bug):
+        if bug['alias']:
```
Comment (marco-c) [inline bugbug/bug_features.py]: All this can be simplified to `return 'CVE' in bug['alias']`.

File: bugbug/bug_features.py [hunk 2]
```diff
@@ -183,6 +183,14 @@ def __call__(self, bug):
         return bug['priority']
 
 
+class bug_has_alias(object):
```
Comment (marco-c) [inline bugbug/bug_features.py]: Please rename this to `bug_has_cve_in_alias`.

**Comment by marco-c** [comment]
Could you check if this makes any difference in the `bug` model?

**Comment by Aftaab99** [comment]
There was no change in the cross-validation scores in the bug model upon adding this feature.

File: bugbug/models/bug.py [hunk 3]
```diff
@@ -33,6 +33,7 @@ def __init__(self, lemmatization=False):
             bug_features.patches(),
             bug_features.landings(),
             bug_features.title(),
+            bug_features.bug_has_cve_in_alias(),
```
Comment (marco-c) [inline bugbug/models/bug.py]: Don't add it here if it didn't make any difference

File: bugbug/bug_features.py (hunk 1 above)
Comment (marco-c) [reply bugbug/bug_features.py]: You should also add some tests for this

**Comment by marco-c** [comment]
Looks good, but there is still one review comment open.

**Comment by Aftaab99** [comment]
> Looks good, but there is still one review comment open.

Are you referring to the one where you told me to to replace the function body with `return 'CVE' in bug['alias']`? It gave me a error as alias is `None` for some blocks hence I added that `if` as a check before that statement.
Or was it any other review comment?

**Comment by marco-c** [comment]
> Are you referring to the one where you told me to to replace the function body with return 'CVE' in bug['alias']? It gave me a error as alias is None for some blocks hence I added that if as a check before that statement.

Yes, that was it. If bug['alias'] can be None, then you can do:
`return bug['alias'] is not None and 'CVE' in bug['alias']`

**Pull request merged**